ament_index_cpp/include/ament_index_cpp/get_resource.hpp

```diff
@@ -21,7 +21,15 @@
 
 namespace ament_index_cpp
 {
-
+/// Get a the content and path of a resource
+/**
+ * \param resource_type type of the resource
+ * \param resource_name name of the resource
+ * \param content content of the resource
```

Comment:
recommend [`\param[out]`](http://www.doxygen.nl/manual/commands.html#cmdparam) to indicate the string is set to the content of the resource (same comment for prefix_path below)